data/constants/mathematical_hoes.json

```diff
@@ -15,6 +15,9 @@
         "THEORETICAL_HOE_CARROT_1",
         "THEORETICAL_HOE_CARROT_2",
         "THEORETICAL_HOE_CARROT_3",
+        "THEORETICAL_HOE_WARTS_1",
+        "THEORETICAL_HOE_WARTS_2",
+        "THEORETICAL_HOE_WARTS_3",
```

Comment:
This list is for the block clicks right?
Why do we block clicks on t3 hoes

Reply:
No idea tbh

Reply:
Also file name is a bit misleading honestly

Reply:
which one

Reply:
Should rather be smth like "block_hoe_clicks.json" or smth

Reply:
I mean tbh it's literally a file with mathematical hoes